httpserver/service_node.h

```diff
@@ -45,13 +46,22 @@ using message_ptr = std::shared_ptr<message_t>;
 
 class Swarm;
 
+struct snode_stats_t {
+
+  // how many times a single push failed
+  uint64_t relay_fails = 0;
```

Comment:
Could be nice to have a counter for the total number of requests/relays performed, to allow expressing the failure count as a percentage?

Reply:
Yeah, I thought about that too. I will extend this struct in subsequent PRs once I decide what other info I want to record. This PR was meant to get things going.

Reply:
Sounds good!